readme.md

```diff
@@ -130,13 +130,17 @@ Type: `Uint8Array | ArrayBuffer`
 
 A buffer representing file data. It works best if the buffer contains the entire file. It may work with a smaller portion as well.
 
-### fileTypeFromFile(filePath)
+#### options
+
+See [file type-options](#file-type-options).
```

Comment:
We don't need this for every method. It's implied that it's the options from the options section.